app/views/categories/index.html.erb

```diff
@@ -0,0 +1,19 @@
+<header>
+<h1 class="category-title">Categories</h1>
+</header>
+
+<main>
+ <% if @categories.any? %>
+  <% @categories.each do |category| %>
+  <%= link_to render(category), user_category_transactions_url(current_user, category.id), class: 'category_wrapper'%> 
```

Comment:
Please follow the design guidelines for font family and font sizes.
Please follow the design for the transactions page, so the header is very different in design than yours. There is a hamburger menu on the left and a search icon on right.

![transaction_page](https://user-images.githubusercontent.com/35504149/190509551-49307dfa-73d1-4993-b02a-7bbcb1a0caee.png)